src/app.coffee

```diff
@@ -62,6 +63,9 @@ class App
                     deviceName: deviceName
                     password: password
                 Devices.registerDevice options, next
+                filterManager = new FilterManager cozyUrl, deviceName, password
+                filterManager.createOrUpdate {file: true, folder: true}, ->
```

Comment:
if I write this:

    filterManager.createOrUpdate file: true, folder: true, ->

I don't know if folder: true, is first or second parameter no ?